### PR TITLE
Seabeta 413 new epc hint design

### PR DESCRIPTION
**Pull request by CodrinCotarlanSoftwire**
Changed hints to blue inset text; Refactored question flow code; Added epc hints to most pages that can use it.

### CODE REVIEW

File: SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml [hunk 1]
```diff
@@ -23,49 +23,49 @@
...
+                        </p>
+                        <p class="govuk-body">
+                            It is accessible if an installer could easily get into it to do some work – for example, if there is an existing loft hatch and some of the space is big enough for someone to stand up in it.
+                        </p>
+                        <p class="govuk-body">
+                            If all the space under your pitched roof, or nearly all the space, is used as living space with finished walls and ceilings then you don’t have an accessible loft.
+                        </p>
+                     </text>;
+            }
+
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml:42]: Required for nested <text> tags

File: SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml [hunk 2]
```diff
@@ -23,49 +23,49 @@
...
                     {
-                        Html = @<text>
-                                   <h1 class="govuk-fieldset__heading">
-                                       Do you have an accessible loft area that hasn’t been converted into a room? 
-                                   </h1>
-                                </text>,
-                        Classes = "govuk-fieldset__legend--l"
+                        Text = "Do you have an accessible loft area that hasn’t been converted into a room?",
+                        Classes = "govuk-fieldset__legend--l",
+                        IsPageHeading = true
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml:50]: As per https://design-system.service.gov.uk/patterns/question-pages/#start-by-asking-one-question-per-page
Single question pages use the legend as heading

File: SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml [hunk 3]
```diff
@@ -23,49 +23,49 @@
...
                 }
                 ))
 
-            <button class="govuk-button" data-module="govuk-button">
-                Continue
-            </button>
+            @await Html.GovUkButton(new ButtonViewModel
+            {
+                Text = "Continue"
+            })
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml:68]: Changed this all around the pages

File: SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml (hunk 2 above)
Comment (DanCorderSoftwire) [reply SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml:50]: Nice

File: SeaPublicWebsite/Views/EnergyEfficiency/CavityWallsInsulated.cshtml [hunk 4]
```diff
@@ -27,17 +27,59 @@
...
+                           </p>
+                       }
+                     </text>;
+                
+                Func<object, object> epcHint = null;
+                if (Model.Epc?.CavityWallsInsulated is not null)
+                {
+                    epcHint = 
+                        @<p class="govuk-body">
+                            Your @Model.Epc?.LodgementDate Energy Performance Certificate suggests you have
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/CavityWallsInsulated.cshtml:62]: I think LodgementDate here is supposed to just be the year, will this show the exact date?

File: SeaPublicWebsite/Views/EnergyEfficiency/FloorConstruction.cshtml [hunk 5]
```diff
@@ -84,63 +110,12 @@
...
-                                    {
-                                        <p>
-                                            Your Energy Performance Certificate suggests you have a <b>solid concrete</b> floor.
-                                        </p>
-                                    }
-                                    <p>
-                                        We can generate recommendations based on this information.
-                                    </p>
-                                }
-                                else if (Model.YearBuilt.HasValue)
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/FloorConstruction.cshtml:115]: Are we definitely getting rid of this when there isn't an EPC?

Comment (DanCorderSoftwire) [reply SeaPublicWebsite/Views/EnergyEfficiency/FloorConstruction.cshtml:115]: Similar question on all the other similar hints on other pages

File: SeaPublicWebsite/Views/EnergyEfficiency/CavityWallsInsulated.cshtml (hunk 4 above)
Comment (DanCorderSoftwire) [reply SeaPublicWebsite/Views/EnergyEfficiency/CavityWallsInsulated.cshtml:62]: You can add a readonly computed property to the EPC class to just get the year if necessary
The LodgementDate in the EpcDto is a string, should it be a DateTime?

File: SeaPublicWebsite.Data/Migrations/20220613140913_UpdateLogdementDateTypeForEpc.cs [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                "LodgementDate",
+                "Epc");
+            migrationBuilder.AddColumn<DateTime>(
+                "LodgementDate",
+                "Epc",
+                type: "timestamp with time zone",
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite.Data/Migrations/20220613140913_UpdateLogdementDateTypeForEpc.cs:18]: Type of the date in postgre

**Review by DanCorderSoftwire — APPROVED** [review]

**Pull request merged**